forage-android/build.gradle

```diff
@@ -97,6 +107,14 @@ dependencies {
     // VGS Collect SDK
     api 'com.verygoodsecurity:vgscollect:1.7.3'
 
+    // Basis Theory SDK
+    implementation ('com.github.basis-theory:basistheory-android:2.1.1') {
+        // Unable to build without excluding this dependency
+        // based on advice from this thread: https://github.com/gradle/gradle/issues/3065#issuecomment-341418873
+        exclude group: 'javax.ws.rs'
+    }
+    implementation "javax.ws.rs:javax.ws.rs-api:2.1@jar"
```

Comment:
`com.github.basis-theory:basistheory-java:0.5.0` is depending on `javax.ws.rs-api`.

The last time I integrated BT I didn't have this issue, but it was on version 1.4. Is this expected from Basis Theory?

Reply:
Unsure, but I was unable to import any versions of the BT SDK without excluding this dependency. I followed the advice found in the thread commented to fix this and whatever code path we are using through BT doesn't seem to rely on this package.

BT had to make multiple updates and new releases to be compatible with our app in the first place.